clients/remote/remote.go

```diff
@@ -67,10 +68,71 @@ func (c Client) Create(ctx context.Context, params *CreateParams) error {
 	if err != nil {
 		return fmt.Errorf("failed to create remote connection %q: %w", params.Name, err)
 	}
+
 	// print confirmation of new connection
 	return c.printRemote(printRemoteOpts{remote: &res})
 }
 
+type ListParams struct {
+	Name      string
+	OrgID     string
+	OrgName   string
+	RemoteURL string
+}
+
+func (c Client) List(ctx context.Context, params *ListParams) error {
+
+	if params.OrgID == "" && params.OrgName == "" && c.ActiveConfig.Org == "" {
```

Comment:
@lesam added some helper functions in #272 that I think you could use here -> line 104